src/hb-face.cc

```diff
@@ -354,12 +365,13 @@ hb_face_is_immutable (const hb_face_t *face)
 
 /**
  * hb_face_reference_table:
- * @face: a face.
- * @tag:
- *
+ * @face: A face object
+ * @tag: The #hb_tag_t of the table to query
```

Comment:
I think we need to document that `HB_TAG_NONE` can be used to fetch the entire font data (but this might not always be possible if the face was created with `hb_face_create_for_tables()` and the reference function does not support this).